src/pykeen/pipeline/api.py

```diff
@@ -915,6 +915,15 @@ def pipeline(  # noqa: C901
     if evaluator_kwargs is None:
         evaluator_kwargs = {}
     evaluator_kwargs = dict(evaluator_kwargs)
+    if evaluator_kwargs.get('filtered'):
+        if use_testing_data and validation is not None:
+            logging.info(
+                "Because we evaluate on the test set, validation triples are added to the set of known positive triples"
+                "which are filtered out when performing filtered evaluation following the approach described by"
+                "(Bordes et al., 2013).",
+            )
+            evaluator_kwargs['additional_filter_triples'] = validation.mapped_triples
```

Comment:
what should happen if they're already set by the user? this will clobber an existing value

Reply:
As mentioned before, in the pre-defined workflows we stick to the standard training/evaluation procedures. Therefore, it would be important that we add a warning/info message notifying that we overwrite this field.

Reply:
Fixed by @lvermue in da0fa8c7